docs/docs/behaviors/mod-morph.md

```diff
@@ -86,3 +86,10 @@ For example, the following configuration morphs `LEFT_SHIFT` + `BACKSPACE` into
     };
 };
 ```
+
+:::note Karabiner-Elements (macOS) and interference with Mod-Morph
```

Comment:
```suggestion
:::note[Karabiner-Elements (macOS) interfering with mod-morphs]
```
We are trying to stick to the [new admonitions syntax](https://docusaurus.io/docs/migration/v3#admonitions-option) to prevent future issues.